packages/evolution-common/src/services/widgets/PathConfig.ts

```diff
@@ -0,0 +1,32 @@
+import { ValidationFunction } from './WidgetConfig';
+import { ParserFunction } from '../parsers/ParserFunction';
+
+/**
+ * Path config include custom admin validations,
+ * flag to include the path in export files or not,
+ * and parsers for custom changes to data before export
+ * Keys are paths with uuids replaced by '@' and array indexes
```

Comment:
There is no keys in this type, maybe add another type that describe the object where you will use the PathConfig type, with the info for the keys there